src/modules/lua.c

```diff
@@ -73,11 +73,17 @@ mtev_lua_lmc_alloc(mtev_dso_generic_t *self, mtev_lua_resume_t resume) {
 void
 mtev_lua_lmc_free(lua_module_closure_t *lmc) {
   if(lmc) {
+    if(lmc->lua_state) lua_close(lmc->lua_state);
```

Comment:
probably don't.  I imagine they existed when mtev_hash_table's weren't thread-safe.  I was just conforming to using the locks to protect the hash table from concurrent manipulation as is done elsewhere.  I can look at removing the locks as unnecessary in a subsequent changeset.